src/main/java/frc/robot/sensors/AnalogSelector.java

```diff
@@ -0,0 +1,120 @@
+package frc.robot.sensors;
+
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.Sendable;
+import edu.wpi.first.wpilibj.smartdashboard.SendableBuilder;
+import edu.wpi.first.wpilibj.smartdashboard.SendableRegistry;
+
+public class AnalogSelector implements Sendable {
+
+    private static final double MODE_ONE_LOW_VOLTAGE = 2.5;
+    private static final double MODE_ONE_HIGH_VOLTAGE = 2.75;
+
+    private static final double MODE_TWO_LOW_VOLTAGE = 2.75;
```

Comment:
A thought: Do we feel that having the same end point and starting point for two adjacent ranges may lead to confusion if the voltage is exactly at that value? i.e. if the voltage is 2.75, is it Mode 1 or Mode 2?  Unless Ranges are defined `Low <= x < High` as opposed to `Low <= x <= High`.

Reply:
The isWithin method used to sort voltage into ranges uses `min <= number && number < max` so I think having the same end point as a starting point for ranges should be ok.